docs/docusaurus.config.js

```diff
@@ -147,6 +149,7 @@ module.exports = {
     [
       "@docusaurus/preset-classic",
       {
+        debug: true,
```

Comment:
This seems mistakenly added

Reply:
Not mistakenly added, but mistakenly not removed yet haha, lemme fix that.

Reply:
Fixed